src/legacy/core_plugins/kibana/public/table_list_view/_table_list_view.scss

```diff
@@ -0,0 +1,8 @@
+@include euiBreakpoint('xs', 's') {
+  .tableListView__table--hideActionsOnMobile {
+    /* Hide action cells on mobile when that property was specified. */
+    .euiTableRowCell--hasActions {
```

Comment:
Please don't override or use `.eui__` classes. In fact, EuiInMemoryTable supports a `mobileOptions` object on each column where you can then set `mobileOptions.show = false`. See the **Responsive** section of https://elastic.github.io/eui/#/display/tables.

Reply:
I was unsure about that two, but since a quick searched showed we're overwriting eui classes in 145 places, I thought it's fine to do :D Also didn't find that option, so fed will use that instead.